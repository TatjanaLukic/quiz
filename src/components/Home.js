import React from 'react';
import '../index.css';

const Home = () => {
    return (
        <div className="Homepage">
           <p> Prepare for job interview!</p>
           <p> Test your knowledge of JavaScript, CSS and React.</p> 
           <p> Click on links above to access quiz.</p>
        </div>
    )
}

export default Home;
