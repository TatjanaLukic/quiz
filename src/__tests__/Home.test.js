import React from 'react';

import { configure, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import Home from '../components/Home';

configure({adapter: new Adapter()});

describe('<Home/>',() => {

it("spanshot matches", () => {
    const tree = shallow(<Home />);
    expect(toJson(tree)).toMatchSnapshot();
  });

});