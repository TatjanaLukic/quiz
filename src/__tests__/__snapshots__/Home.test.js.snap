// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home/> spanshot matches 1`] = `
<div
  className="Homepage"
>
  <p>
     Prepare for job interview!
  </p>
  <p>
     Test your knowledge of JavaScript, CSS and React.
  </p>
  <p>
     Click on links above to access quiz.
  </p>
</div>
`;
